frontend/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
```

Comment:
I guess you can remove the local `.gitignore`-file because it's handled by the `.gitignore` in the root-directory.